lib/apiservers/engine/backends/container.go

```diff
@@ -934,347 +967,6 @@ func (c *Container) ContainerAttach(name string, ca *backend.ContainerAttachConf
 	return err
 }
 
-//-------------------------------------
```

Comment:
Moving it is described in the PR description.  Had to move to our new container_proxy, which implements an interface that makes unit test mocking possible.  Any operation that needs to be mocked has to go into that file and function proto added to the interface.  That also includes utility function it relies on.